src/test/java/com/sonyericsson/jenkins/plugins/bfa/PluginImplHudsonTest.java

```diff
@@ -98,53 +96,6 @@ public void testGetKnowledgeBaseDescriptors() throws Exception {
         assertTrue("Expected to find a DifferentKnowledgeBaseDescriptor", foundDifferentKb);
     }
 
-    /**
-     * Tests {@link PluginImpl#configure(org.kohsuke.stapler.StaplerRequest, net.sf.json.JSONObject)}. with the same
-     * KnowledgeBase as before.
-     *
-     * @throws Exception if so.
-     */
-    @Test
-    public void testConfigure() throws Exception {
```

Comment:
Why are these removed?

Reply:
I "think" it was because this test was testing form binding and form binding is now using databound semantics and tested via the jcasc test, re-testing databinding here would be awkward.